ModulemdTranslationHelpers/Utils.py

```diff
@@ -108,7 +108,10 @@ def get_index_from_tags(session, tags):
     return index
 
 
-def get_translation_catalog_from_index(index, project_name):
+def get_translation_catalog_from_index(index, project_name, debug):
+    if debug == True:
```

Comment:
Don't add a `debug` parameter here, just have the `cli()` function set `logging.setLevel(logging.DEBUG)` globally. Then when you run with `--debug`, it'll be printed to the screen.